components/studentActivity.js

```diff
@@ -15,7 +15,8 @@ export default function getStudentActivity(props) {
   }
   var mostRecentDate = new Date(mostRecentCompletionTime);
   let mostRecentDateText =
-    'Last completion time: ' + mostRecentDate.toLocaleString();
+    'Last completion time: ' +
+    mostRecentDate.toLocaleString('en-US', { timeZone: 'America/Los_Angeles' });
```

Comment:
Does it make sense to hardcode the timezone to PST for all users of the app?  What if they are in a different time zone?  I'm not opposed to it, but I'm curious what the reasoning was behind this.

Reply:
Hi @utsab
Cc @theGaryLarson @Komal914 

**Summary:** During Continuous Integration (CI) testing without a default locale and time zone, `AM` versus `PM` drifts between @theGaryLarson's local and GitHub's computers happened.

`toLocaleString()` without setting a default locale and default time zone causes inconsistent behaviors between @theGaryLarson (PDT) and GitHub Continuous Integration (UTC) when matching 12-hour clock format `AM` versus `PM` during Jest snapshot testing.

Initially, @theGaryLarson changed
`toLocaleString()`
to
`toLocaleString('en-US', { timeZone: 'UTC' })`
via [[1]](https://github.com/freeCodeCamp/classroom/commit/bdb1a5782aefeee24291a5a6a527d92d4eb2be6a).

Then, @theGaryLarson
changed it to
`toLocaleString('en-US', { timeZone: 'America/Los_Angeles' })`
via [[2]](https://github.com/freeCodeCamp/classroom/pull/422/commits/5bd322fcfd58c786664864dfb57e90f9c2e7a2dc).

In both cases, the newly set default locale is `en-US` for the 12-hour clock format.

I believe @theGaryLarson chose the locale `en-us`, and the 
 time zone `America/Los_Angeles` to match @theGaryLarson's local computer and GitHub's computers to prevent `AM` versus `PM` drifts during CI testing.

References:
1. https://github.com/freeCodeCamp/classroom/commit/bdb1a5782aefeee24291a5a6a527d92d4eb2be6a
2. https://github.com/freeCodeCamp/classroom/pull/422/commits/5bd322fcfd58c786664864dfb57e90f9c2e7a2dc